Circle_Sort/Circle_Sort.php

```diff
@@ -0,0 +1,85 @@
+<?php
+ 
+//In circle sort The elements of the array lying on the same circle diametrically opposite to each other are compared.
+
+function circleSortRec($a, $low, $high)
+{
+    $swapped = false;
+    
+    if($low == $high)
+    {
+        return false;
+    }
+
+    $lo = $low;
+    $hi = $high;
+
+    while($lo < $hi)
+    {
+
+        if($a[$lo] > $a[$hi])
+        {
+            $temp = $a[$lo];
+            $a[$lo] = $a[$hi];
+            $a[$hi] = $temp;
+            $swapped = true;
+        }
+        $lo++;
+        $hi--;
+    }
+      
+    if($lo == $hi)
+    if($a[$lo] > $a[$hi+1])
```

Comment:
Add space around +

Reply:
@somya-kapoor  Done the required changes.